scipy/signal/wigner.py

```diff
@@ -0,0 +1,95 @@
+"""Tools based on the Wigner quasiprobability distributions.
+"""
+
+import numpy as np
+from scipy.fft import fft
+
+
+__all__ = ['wvd']
+
+
+def wvd(x, fs=1.0, resolution=1, win_size=None):
+    """
+    Compute the Wigner-Ville distribution.
+
```

Comment:
Could you add one paragraph here about what the Wigner-Ville distribution is/does?

Reply:
Done.